config.json

```diff
@@ -1178,7 +1194,7 @@
         "name": "Sublist",
         "uuid": "bb00e97f-8dcf-40ba-b8c2-c3cb72bbaf97",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["array-analysis", "comparison"],
```

Comment:
Needs `conditionals` as well, I didn't see a solution without an `if` statement. The inputs are arrays so might be good to add `arrays`.